### ISSUE TITLE
`server.install` does not work in `SolidStart` site

### ISSUE THREAD

**Issue by ian-pascoe** [sst/ion#1110]
When I try to deploy my solidstart site and add installs to the server function, they are not added.

To reproduce:
1. Use SolidStart example
2. Add `server.install` option
3. Check deployed code and see that install was not executed

**Comment by fwang** [comment]
You are right. `server.install` has no effect on SolidStart sites. `SolidStart` uses Nitro behind the scene. Nitro packages the lambda code into `.output/server`. And `SolidStart` uses it as is.

What packages are you trying to bundle into the server lambda code?

**Comment by ian-pascoe** [comment]
I am trying to have sentry packages installed for instrumentation. The sentry docs want your sentry init file to be called via `NODE_OPTIONS=‘—import instrumentation.mjs’` which makes it get shaken during build

**Comment by fwang** [comment]
Can u check if it can be configured in `app.config.ts` to include specific npm packages in the server build output?

/cc @thdxr do u know if it's possible?

**Comment by ian-pascoe** [comment]
I have tried several config options but it keeps getting shaken. I am about to go the lambda layer route if I can’t figure it out

**Comment by fwang** [comment]
Just talked to @thdxr, he's working on some stuff in this area. He will fix this.

For context, SolidStart's server output looks like this:
<img width="145" alt="Screen Shot 2024-09-21 at 3 45 08 PM" src="https://github.com/user-attachments/assets/7e235a52-2920-497a-98ca-8b1cb414a9dd">

We could inject `server.install` in `package.json`, and run `npm install`.
